ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/InfoMapper.java

```diff
@@ -266,17 +298,18 @@ private static OpenAPIInfo updateOpenAPIInfoModel(SeparatedNodeList<MappingField
                     fieldValue = expressionNode.toString().trim().replaceAll("\"", "");
                     if (!fieldValue.isBlank()) {
                         switch (fieldName) {
-                            case CONTRACT:
-                                infoBuilder.contractPath(fieldValue);
-                                break;
-                            case TITLE:
-                                infoBuilder.title(fieldValue);
-                                break;
-                            case VERSION:
-                                infoBuilder.version(fieldValue);
-                                break;
-                            default:
-                                break;
+                            case CONTRACT -> infoBuilder.contractPath(fieldValue);
+                            case TITLE -> infoBuilder.title(fieldValue);
+                            case VERSION -> infoBuilder.version(fieldValue);
+                            case EMAIL -> infoBuilder.email(fieldValue);
+                            case DESCRIPTION -> infoBuilder.description(fieldValue);
+                            case CONTACT_NAME -> infoBuilder.contactName(fieldValue);
+                            case CONTACT_URL -> infoBuilder.contactURL(fieldValue);
+                            case TERMS_OF_SERVICE -> infoBuilder.termsOfService(fieldValue);
+                            case LICENSE_NAME -> infoBuilder.licenseName(fieldValue);
+                            case LICENSE_URL -> infoBuilder.licenseURL(fieldValue);
+                            default -> {
+                            }
                         }
                     }
```

Comment:
I think this can have an early return so that we could remove unwanted indentation.
```java
if (fieldValue.isBlank()) {
  // return from here
}
switch (fieldName) {
    case CONTRACT -> infoBuilder.contractPath(fieldValue);
    // ...
}
```